From 17557601f0570797a7398ed1dc7db733e6503658 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 8 Jan 2024 10:03:49 +0100
Subject: [PATCH 0372/1695] dts: stm32: enable rtc on stm32mp157c-exx

Enable RTC in stm32mp157c-ed1.dts.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e49c7dbbd85e86d3c063f8326374ac157b1343ea dts: stm32: enable rtc on stm32mp157c-exx]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: Ic3c092b00d49be14daed2aa9267776420b4e34a4
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/349694
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp157c-ed1.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index 74797471e..d19aba921 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -484,7 +484,7 @@
 };
 
 &rtc {
-	status = "disabled";
+	status = "okay";
 };
 
 &sdmmc1 {
-- 
2.47.2

